src/main/java/com/example/demo/exceptions/ExceptionHandleControllerAdvice.java

```diff
@@ -110,4 +110,16 @@
 
     	return new ResponseEntity<>(body, HttpStatus.UNAUTHORIZED);    	
     }
+
+    @ExceptionHandler(UnallowedOperationException.class)
+    public ResponseEntity<Object> handleUnallowedOperationException(UnallowedOperationException ex,WebRequest request){
+    	
+        Map<String, Object> body = new LinkedHashMap<>();
+        body.put("timestamp", LocalDateTime.now());
+        body.put("message", ex.getMessage());
+        if(!ex.getErrors().isEmpty())
+        	body.put("errors", ex.getErrors());
+
+    	return new ResponseEntity<>(body, HttpStatus.FORBIDDEN);
```

Comment:
we have similar code all over this ControllerAdvice
can we extract to formulation: `return buildResponse(exception, HttpStatus.FORBIDDEN);`